app/services/notifications/milestone/send.rb

```diff
@@ -0,0 +1,65 @@
+module Notifications
+  module Milestone
+    class Send
+      def initialize(milestone_hash)
```

Comment:
A couple of suggestions about the service:
- pass an article and type explicitly instead of a hash, in my opinion, it would be easier to understand (+ slightly less code)
- `next_milestone` could be just a private method in the service object because it is used only here, for now, there's no need to reuse it in other places. It would be a bit easier to understand and there would be no need to pass `milestone_hash` 
- it would be nice to document arguments passed to the service like in: https://github.com/thepracticaldev/dev.to/blob/e639f376ee088739909446515901afc051cc91e3/app/services/notifications/new_follower/send.rb#L5-L8

Reply:
I have updated the code in this way.